pom.xml

```diff
@@ -17,19 +17,30 @@
     <version>5.2.0</version>
```

Comment:
Following the same strategy a `pom-omero-client` thoughts about bumping this to 5.2.1-SNAPSHOT?